Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -634,6 +634,10 @@ Ref<MutableStyleProperties> ViewTransition::copyElementBaseProperties(Element& e
         if (!container)
             break;
         LayoutSize containerOffset = renderer->offsetFromContainer(*container, LayoutPoint());
+        if (container->isRenderView()) {
+            auto frameView = renderer->view().protectedFrameView();
+            containerOffset -= toLayoutSize(frameView->scrollPositionRespectingCustomFixedPosition());
+        }
```

Comment:
Does it work within scrollers?

Reply:
It does! Just the FrameView scrolling is special.